geonode/maps/templates/leaflet_maps/map_edit.html

```diff
@@ -55,8 +55,8 @@
                         <label for="add_layer_select">Available layers</label>
                         <select class="form-control" id="add_layer_select">
                             {% for layer in layers %}
-                                <option value="{{ layer.typename }}"
-                                        id="option_{{ layer.typename }}">{{ layer.title }}</option>
+                                <option value="{{ layer.alternate }}"
```

Comment:
Upstream changes. Field name changes, doesn't get refactored because it is in template.